app/models/school.rb

```diff
@@ -184,8 +184,9 @@ class ProcessDataError < StandardError; end
   #are archived, with chance of returning if we receive funding
   scope :active,              -> { where(active: true) }
   scope :inactive,            -> { where(active: false) }
-  scope :deleted,             -> { inactive.where.not(removal_date: nil)}
-  scope :archived,            -> { inactive.where(removal_date: nil)}
+  scope :deleted,             -> { inactive.where.not(removal_date: nil) }
+  scope :archived,            -> { inactive.where(removal_date: nil) }
+  scope :active_and_archived, -> { find_by_sql(active.to_sql + ' UNION ALL ' + archived.to_sql) }
```

Comment:
Is this the same as writing active.or(archived) ? Just to simplify a bit if so...